examples/vm-cirros-cluster-csmall.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   instancetype:
     kind: VirtualMachineClusterInstancetype
-    name: csmall
+    name: cluster-csmall
```

Comment:
You actually need to update the following code and then run `make generate` that in turn generates the examples :

https://github.com/akrejcir/kubevirt/blob/cd0a28c2c34f5e447ff2d3465ca5eb2708feae66/tools/vms-generator/utils/utils.go#L1261-L1264

More details below:

https://github.com/kubevirt/kubevirt/blob/main/docs/getting-started.md#automatic-code-generation

Reply:
Done. Hope everything is fine now.

Reply:
ACK thanks, I'll just let `make generate` run in CI before giving my `/lgtm`